### DISCUSSION TITLE
What is "prior"?

### DISCUSSION THREAD

**Discussion by EdwardYuyangTian** [cornellius-gp/gpytorch#2051] [Q&A]
I'm really a beginner of GP. Sorry for having to ask this, but what does "prior" (in the basic tutorial of Hyperparameters) mean, and what's the use of it?

Thanks a lot!

**Comment by gpleiss** [comment]
Here "prior" has the standard Bayesian meaning: adding a prior to a hyperparameter treats hyperparameters as latent random variables with a prior belief. See section 5.2 of Rasmussen and Williams' book (Gaussian processes for machine learning).

If you are unfamiliar with this concept I also suggest reading chapters 1 and 2 of the Rasmussen and Williams book.
